src/Temporalio.Extensions.OpenTelemetry/README.md

```diff
@@ -53,6 +53,45 @@ var client = await TemporalClient.ConnectAsync(new("localhost:7233")
 // Now client and worker calls are traced...
 ```
 
+If you are using dependency injection with `Temporalio.Extensions.Hosting`, you have to configure the client and
+worker separately. For example,
+
+```csharp
+// In Program.cs
+using Temporalio.Extensions.Hosting;
+using Temporalio.Extensions.OpenTelemetry;
+
+// Setup the tracer provider
+using var tracerProvider = Sdk.CreateTracerProviderBuilder().
+    AddSource(
+        TracingInterceptor.ClientSource.Name,
+        TracingInterceptor.WorkflowsSource.Name,
+        TracingInterceptor.ActivitiesSource.Name).
+    AddConsoleExporter().
+    Build();
+
+// Setup dependency injection
+var builder = WebApplication.CreateBuilder(args);
+builder.Services
+    // Create a client to localhost on "default" namespace with the tracing
+    // interceptor
+    .AddTemporalClient(opts =>
+        {
+            opts.TargetHost = "localhost:7233";
+            opts.Namespace = "default";
+            opts.Interceptors = new[] { new TracingInterceptor() };
+        })
```

Comment:
Many users don't even configure a client in their worker processes when setting up with generic host support. I wonder if we should make that clear.

Reply:
Hm, I have two interpretations of what you're suggesting to clarify:

* What this client setup code is doing
* How a configured client affects the worker process

Though in either case, I think it isn't this README's job to explain generic host support. If it's clear to users that they should look at https://github.com/temporalio/sdk-dotnet/blob/main/src/Temporalio.Extensions.Hosting/README.md for that, I think we've done our job here. If you think _that_ isn't clear, I'm happy to add the pointer :)

Reply:
> Though in either case, I think it isn't this README's job to explain generic host support

Right, don't need to explain generic host support, it's just a bit confusing to see `AddTemporalClient` and `AddHostedTemporalWorker` both here. Either remove the former or make the latter use the overload that works with the former.